app/src/main/java/model/place/Category.kt

```diff
@@ -17,48 +17,47 @@
 package com.guerinet.mymartlet.model.place
 
 import android.content.Context
-import androidx.room.Entity
-import androidx.room.PrimaryKey
+import com.google.firebase.firestore.DocumentSnapshot
 import com.guerinet.mymartlet.R
 import java.util.Locale
 
 /**
  * A type of place that the user can filter by
  * @author Julien Guerinet
  * @since 1.0.0
- *
- * @property id Category Id
- * @property en Category name in English
- * @property fr Category name in French
  */
-@Entity
-data class Category(
-    @PrimaryKey var id: Int = 0,
-    var en: String,
-    var fr: String
-) {
+class Category() {
```

Comment:
Firestore required a public no-args constructor to map their documents to our objects.

Reply:
You can also use `@JvmOverloads` to create empty and full constructors, provided you already have default values

Reply:
Given that in these cases we are pretty much only using the no-arg constructors anyway (Except for the ALL category, which needs an entirely different constructor), I don't see a real need for using `@JvmOverloads`. Also using that would create a bunch of unnecessary constructors depending on the number of arguments.

Reply:
I can see that so long as we are staying with firebase